elf/module.go

```diff
@@ -457,6 +458,10 @@ func (b *Module) closeMaps() error {
 	return nil
 }
 
+type CloseOptions struct {
```

Comment:
Exported types should be documented, I think it is requested by `go fmt` or `lint`.